testing/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+load("@rules_jvm_external//:defs.bzl", "artifact")
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+
+package(default_visibility = ["//visibility:public"])
+
+kt_android_library(
```

Comment:
How about if we put comments here to describe `kt_android_library`, its binding of source files in the utility module? WDYT?

Reply:
Sure!

Reply:
I added a short comment - let me know if you think i should add more detail!

Reply:
For now, it's workable.